.github/workflows/release-docker.yml

```
@@ -0,0 +1,45 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
```

Could you please use the following format:

```suggestion
      - name: Checkout
        uses: actions/checkout@v2
```